api/app/src/Service/RequestIdLoggerProcessor.php

```diff
@@ -32,30 +33,56 @@ public function __construct(ContainerInterface $container)
     }
 
     /**
-     * Add request header 'x-request-id' into ['extra']['request_id']
+     * Add request header 'x-aws-request-id' into ['extra']['aws_request_id']
      * Does not change the record if the scope is not active, or the request is not found or doesn't contain the header.
      *
      * @return array same record with extra info
      */
     public function processRecord(array $record)
     {
         $reqId = self::getRequestIdFromContainer($this->container);
```

Comment:
I'd suggest getting the request_stack once, then passing it to the static methods if it exists; otherwise don't bother calling them at all.

Reply:
Good idea. Done